File: Cine/gestorAplicación/cine/Sala.java
package gestorAplicación.cine;

import java.io.*;

import java.util.HashMap;

import java.util.Map;


public class Sala implements Serializable {
	
	private static final long serialVersionUID = 1L;
	
    private String nombre; // nombre de la sala, por ejemplo Sala 1
    // lista de asientos ocupados
    private Map<Pelicula, Horario> cartelera = new HashMap<Pelicula, Horario>(); // Cartelera de la sala | Pelicula, Horario
    private Map<String, boolean[] > Asientos = new HashMap<String, boolean[]>();
//constructores
    public Sala ( Map<String, boolean[]> Asientos, Map<Pelicula, Horario> cartelera) {
    	this.nombre = "Sala Unal";
    	this.Asientos = Asientos;
        this.cartelera = cartelera;

        
    }
   
    public Sala (String nombre, Map<String, boolean[]> Asientos, Map<Pelicula, Horario> cartelera) {
        this.nombre = nombre;
        this.Asientos = Asientos;
        this.cartelera = cartelera;

    }
//metodos get y set
    public String getNombre() {
    	
        return nombre;
        
    }

    public void setNombre(String nombre) {
    	
        this.nombre = nombre;
        
    }

    public Map<Pelicula, Horario> getCartelera() {
        return cartelera;
        
    }

    public void setCartelera(Map<Pelicula, Horario> cartelera) {
        this.cartelera = cartelera;
        
    }

    public Map<String, boolean[] > getAsientos() {
        return Asientos;
        
    }

    public void setAsientos(Map<String, boolean[] > Asientos) {
        this.Asientos = Asientos;
        
    }
    //print the names of the movies and  the times in cartelera
}

File: Python/src/gestionAplicacion/cinepy/Sala.py
class Sala:
    def __init__(self, nombre, asientos, cartelera):
        self.nombre = nombre
        self.asientos = asientos
        self.cartelera = cartelera


    def getNombre(self):
        return self.nombre

    def getAsientos(self):
        return self.asientos

    def getCartelera(self):
        return self.cartelera

    def setNombre(self, nombre):
        self.nombre = nombre

    def setAsientos(self, asientos):
        self.asientos = asientos

    def setCartelera(self, cartelera):
        self.cartelera = cartelera

    def __str__(self):
        return self.nombre 

    